### ISSUE TITLE
Plants and Trees "instantly" Mature on server restart

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#6826]
Any plant, wild, planted crops and trees are all mature after a server restart even if just planted before the server is shutdown.

Repro:  Identify newly sprouted trees and mark them.  Plant several crops.  Save world.  Immediately restart server.  Go back and check stats of plants and marked trees.  They will have suddenly become mature plants/trees.

**Comment by Tetramorium** [comment]
Happens on my server too. Throws the economy out of balance for servers that frequently have to restart.

**Comment by SofaEco** [comment]
same here 7.3.1 brought this up to our server

**Comment by Last2Night** [comment]
Same problem

**Comment by Arti-llery** [comment]
Yep, have had this issue quite a bit. Gave everyone a nice starting food boost, but still, shouldn't be happening.
